### PR TITLE
Fix various issues

### PR DESCRIPTION
**Pull request by benthamite**
- Amend wrong curly quotes
- Turn straight quotes into curly quotes
- Make all blockquotes consistent (some had quotes, others didn’t; some had only
  closing quotes)
- Restore narrow non-breaking space in a few cases
- Fix a few typos
- Correct some translations
- Position a few footnotes correctly (after punctuation mark rather than before it)
- Use narrow space as thousands separator in numerals

### CODE REVIEW

**Comment by benthamite** [comment]
@whyboris, I’ll answer your email shortly, but I wanted to first fix the issues that could be addressed by modifying the source files. The comments in my email will point to stuff originating in your code, that I cannot address myself. Thanks!

**Comment by whyboris** [comment]
Thank you very much for improvements 🥳 🤝 😁 
I'm happy to try to fix things you mention over email 👍 
Let me know whenever you'd like me to merge this PR in -- no rush 🙂

**Review by whyboris — APPROVED** [review]
Thank you for all the improvements 🙇‍♀️

File: content/es/acting-on-utilitarianism.md [hunk 1]
```diff
@@ -7,9 +7,7 @@ draft: false
 menu: "acting"
 weight: 5
 ---
-
-> _Crea toda la felicidad que seas capaz de crear; elimina toda la miseria que seas capaz de eliminar. Cada día te permitirá, te invitará a añadir algo al placer de los demás, o a disminuir algo de sus penas."_ \
-> \- [Jeremy Bentham](https://altruismoeficaz.net/autores/jeremy-bentham)[^1]
```
Comment (whyboris) [inline content/es/acting-on-utilitarianism.md:12]: accidentally deleted this line :trollface:

Comment (whyboris) [reply content/es/acting-on-utilitarianism.md:12]: fixed with https://github.com/whyboris/utilitarianism.net/pull/133 👌

**Pull request merged**